Assets/Scripts/SS3D/Systems/Health/BodyParts/HumanTorso.cs

```diff
@@ -58,6 +57,19 @@ protected override void AddInitialLayers()
 
         InvokeOnBodyPartLayerAdded();
     }
+    
+    protected override void InflictDamage(BodyLayer layer, DamageTypeQuantity damageTypeQuantity)
```

Comment:
why is this needed ? remember, the expected behavior for the health system is that the human dies only when its brain dies.